### PR TITLE
I made some css that makes it coding train themed.

### PR DESCRIPTION
**Pull request by GiggioG**
### This is in regard to #1 
I made this css file that makes it coding train themed. Here is what it does: 

- Center things
- Adds 'Cubano' font for h1 tags
- Adds 🚂 before and 🌈 after h1 tags
- Adds gradient background with the coding train colors.
- Removes default margin and padding

From now if you want I can update it after every stream so it works, or leave it for you to decide. That is your choise, that I expect you to answer as a comment to this pull request.
Love your vids, Dan. Best regards - Gigo.

### CODE REVIEW

**Comment by shiffman** [comment]
Wonderful this is terrific! I will gladly merge and will probably wait to do so when I am streaming next so I can show it live. The background however has to stay transparent for it to be an overlay. Though I could put it behind me since I have a greenscreen. You could make the rectangle bar a rainbow gradient though if you want!

**Comment by GiggioG** [comment]
I will make the background white, because if it was green it would catch the part out of the rainbow emoji, but I won't touch the p5 code, because I wanted it to be only a css thing. Thank you for your input -Gigo.

**Comment by GiggioG** [comment]
Change the background-color to rgb(255, 255, 255);

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@font-face {
+    font-family: "cubanoregular";
+    src: url("https://thecodingtrain.com/assets/fonts/cubano-regular-webfont.woff2");
+}
+body {
+    display: flex;
+    flex-direction: column;
+    place-content: center;
+    place-items: center;
+    background-color: rgb(255,255,255);
```
Comment (shiffman) [inline public/style.css:10]:
```suggestion
    background-color: rgb(0,255,0);
```
This is for the chroma-key overlay

**Pull request merged**